@anjakefala `man` page build is broken. Any idea how to fix it?

```
VisiData v2.11dev
widest option+default is "disp_canvas_charset", width 275
using width for cli options of 43
Usage: .No normal_text ... (#760)
Usage: .No normal_text ... (#769)
troff: <standard input>:1149: warning [p 13, 7.2i]: can't break line
troff: <standard input>:1213: warning [p 14, 4.7i, div 'doc-box-dtl', 0.2i]: can't break line
troff: <standard input>:1213: warning [p 14, 4.8i]: can't break line
ul: unknown escape sequence in input: 33, 133
```

I've added man page build to CI so that it can be caught early - https://github.com/saulpw/visidata/runs/8246919290?check_suite_focus=true

The error a a bit different there.
```
VisiData v2.11dev
widest option+default is "disp_canvas_charset", width 275
using width for cli options of 43
Usage: .No normal_text ... (#760)
Usage: .No normal_text ... (#769)
troff: <standard input>:1149: warning [p 13, 7.2i]: can't break line
troff: <standard input>:1213: warning [p 14, 4.7i, div 'doc-box-dtl', 0.2i]: can't break line
troff: <standard input>:1213: warning [p 14, 4.8i]: can't break line
*** stack smashing detected ***: terminated
Usage: .No normal_text ... (#760)
Usage: .No normal_text ... (#769)
dev/mkman.sh: line 44:  2721 Done                    MAN_KEEP_FORMATTING=1 COLUMNS=1000 man "$MAN"/vd.1
      2722 Aborted                 (core dumped) | ul
      2723                       | aha --no-header >> "$manhtml"
Error: Process completed with exit code 134.
```

--- a/visidata/man/vd.inc
+++ b/visidata/man/vd.inc
@@ -859,6 +859,7 @@ show/hide methods and hidden properties
 .No launch vd with Ar subsheet No of Ar toplevel No at top-of-stack, and cursor at Ar col No and Ar row Ns ; all arguments are optional
 .Pp
 .Lo f filetype filetype
+.Lo d delimiter delimiter
 .No "tsv               "
 set loader to use for
 .Ar filetype

